src/completed-tasks/completed-tasks-slice.ts

```diff
@@ -0,0 +1,36 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
This is a core change: this defines all the redux handling for the `completedTasks` part of the redux state